src/lldpd/patch/series

```diff
@@ -1,3 +1,4 @@
 # This series applies on GIT commit 396961a038a38675d46f96eaa7b430b2a1f8701b
 0001-return-error-when-port-does-not-exist.patch
 0002-Let-linux-kernel-to-find-appropriate-nl_pid-automa.patch
+0003-update-tx-interval-immediately.patch
```

Comment:
can you upstream this patch?

Reply:
You mean upstream to https://github.com/vincentbernat/lldpd ?

Reply:
yes